CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 Enhancements:
 - Added .NET Standard/.NET Core support for NLog (@snakefoot, #200)
 - Added .NET Standard/.NET Core support for log4net (@snakefoot, #201)
+- Add .NET Standard 1.5 as additional target to NuGet package. This adds support for `in` parameter modifiers on e.g. .NET Core (@stakx, #339)
```

Comment:
Could to clarify the changelog entry so it is clear this wasn't a problem on .NET Framework. Thanks.

Reply:
I rewrote the changelog entry as follows:

> DynamicProxy supported C# `in` parameter modifiers only on the .NET Framework up until now. Adding .NET Standard 1.5 as an additional target to the NuGet package makes them work on .NET Core, too (@stakx, #339)

Is that OK?

Reply:
Yep